conclusions/rq3.md

```diff
@@ -0,0 +1,5 @@
+# How are requirements established and used in software development?
+
+Section 2 describes the process of requirements engineering. Implementing structured requirements development and management early on can help deal with the challenges of building a large software systems. One of the main challenges is coordinating dependencies between requirements, and can be dealt with using coordination mechanisms.
```

Comment:
`is coordinating dependencies between requirements, and can be dealt with ..` Seems like an odd sentence. Maybe split the sentence like `between requirements. Coordination mechanisms can solve these challenges.`